src/ts/playerutils.ts

```diff
@@ -109,10 +118,15 @@ export namespace PlayerUtils {
 
       // HLS live detection workaround for Android:
       // Also re-evaluate during playback, because that is when the live flag might change.
-      // (Doing it only in Android Chrome saves unnecessary overhead on other plattforms)
+      // (Doing it only in Android Chrome saves unnecessary overhead on other platforms)
       if (BrowserUtils.isAndroid && BrowserUtils.isChrome) {
         player.on(player.exports.PlayerEvent.TimeChanged, liveDetector);
       }
+
+      // DurationChanged event was introduced with player v8.19.1
+      if (player.exports.PlayerEvent.hasOwnProperty('DurationChanged')) {
```

Comment:
This can simply be `if (player.exports.PlayerEvent.DurationChanged) {`

Reply:
ed50331